api/chaos/redisexperimentation/v1/redisexperimentation.proto

```diff
@@ -0,0 +1,67 @@
+syntax = "proto3";
+
+package clutch.chaos.redisexperimentation.v1;
+
+import "validate/validate.proto";
+
+option go_package = "github.com/lyft/clutch/backend/api/chaos/redisexperimentation/v1;redisexperimentationv1";
+
+// The configuration of a Redis fault.
+message RedisFaultConfig {
+  // The targeting of the fault describing what redis requests are being considered for faults.
+  RedisFaultTargeting fault_targeting = 1 [ (validate.rules).message.required = true ];
+
+  oneof fault {
+    option (validate.required) = true;
+
+    // The error fault responds to redis commands with an error.
+    RedisErrorFault error_fault = 2;
+    // The latency fault delays redis commands.
+    LatencyFault latency_fault = 3;
```

Comment:
Do we want to add Redis prefix here or remove it from `RedisErrorFault `? I would go with latter I think

Reply:
Yeah no need for the Redis prefix anymore here. I'll drop.